src/pages/todo/actions.js

```diff
@@ -1,3 +1,8 @@
 import { createRequestTypes } from '../../common/api'
 
 export const LIST_TODOS = createRequestTypes('LIST_TODOS');
+export const MOVE_TODO = createRequestTypes('DELETE_TODOS');
```

Comment:
@vinhkq 
`export const MOVE_TODO = createRequestTypes('DELETE_TODOS');`
should be 
`export const MOVE_TODO = createRequestTypes('MOVE_TODO');`

`export const CREATE_TODO = createRequestTypes('CREATE_TODOS');`
should be
`export const CREATE_TODO = createRequestTypes('CREATE_TODO');`

and so on.